aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -922,6 +923,37 @@ impl AptosVM {
         simulation_vm.simulate_signed_transaction(&state_view.as_move_resolver(), txn, &log_context)
     }
 
+    pub fn execute_view_function(
+        state_view: &impl StateView,
+        module_id: ModuleId,
+        func_name: Identifier,
+        arguments: Vec<Vec<u8>>,
+        type_args: Vec<TypeTag>,
+        gas_budget: u64,
+    ) -> Result<Vec<MoveValue>> {
```

Comment:
Why MoveValue as output and not bcs?

Reply:
There will be two ways to return the value here: json and bcs. `MoveValue` would allow us to convert into either form easier.